### PR TITLE
Add ability to get/set corner label for wxGrid header

### PR DESCRIPTION
**Pull request by p-kalugin**

### CODE REVIEW

File: samples/grid/griddemo.cpp [hunk 1]
```diff
@@ -920,6 +935,73 @@ void GridFrame::SetColLabelVertAlignment( wxCommandEvent& WXUNUSED(ev) )
 }
 
 
+void GridFrame::SetCornerLabelHorizAlignment( wxCommandEvent& WXUNUSED(ev) )
```
Comment (vadz) [inline samples/grid/griddemo.cpp]: Actually after testing this I've realized that the way these menu commands work doesn't make any sense: the "horizontal" and "vertical" alignment are radio items, but selecting them somehow cycles through the horizontal/vertical alignment styles independently??

It looks like the new functions just mimic the existing ones, but I'd rather not do this unless I'm really missing something here.

File: samples/grid/griddemo.cpp [hunk 2]
```diff
@@ -920,6 +935,73 @@ void GridFrame::SetColLabelVertAlignment( wxCommandEvent& WXUNUSED(ev) )
...
+        case wxALIGN_BOTTOM:
+            vert = wxALIGN_TOP;
+            break;
+    }
+
+    grid->SetCornerLabelAlignment( horiz, vert );
+}
+
+
+void GridFrame::ToggleCornerLabelOrientation( wxCommandEvent& WXUNUSED(ev) )
```
Comment (vadz) [inline samples/grid/griddemo.cpp]: In my testing this doesn't seem to work, i.e. I don't get vertical text in the corner. Is it normal?

File: samples/grid/griddemo.cpp (hunk 1 above)
Comment (p-kalugin) [reply samples/grid/griddemo.cpp]: Indeed, corner alignment test just mimics existing similar tests for row and column alignment. I also found them somewhat confusing at first, but then realized, that it is a smart way of testing all alignment combinations with only two radio buttons. Anyway, I probably considered existing tests as 'authoritative' in some sense...

File: samples/grid/griddemo.cpp (hunk 2 above)
Comment (vadz) [reply samples/grid/griddemo.cpp]: Damn, sorry, I must have done something wrong as I've retested it now (this was under MSW FWIW) and it does work. Sorry for the false alert.

File: samples/grid/griddemo.cpp (hunk 1 above)
Comment (vadz) [reply samples/grid/griddemo.cpp]: Well, OK, this is not the highest priority, but it's really weird. It'd be nice to streamline all this somehow later. But for now let's merge this as it is, thanks.

**Comment by vadz** [comment]
FYI: resubmitted to test CI builds as https://github.com/wxWidgets/wxWidgets/pull/928

**Pull request merged**